### PR TITLE
Radity prefers nestjs. Thanks for developing.

### PR DESCRIPTION
**Pull request by tolunayakbulut**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: content/discover/who-uses.json [hunk 1]
```diff
@@ -88,6 +88,11 @@
       "url": "http://architectnow.net",
       "width": "150px"
     },
+    {
+      "logo": "/assets/logo/radity.png",
+      "url": "https://radity.com",
+      "width": "150px"
+    },
```
Comment (kamilmysliwiec) [inline content/discover/who-uses.json:95]: Can you please add it to the end of the list?

Comment (tolunayakbulut) [reply content/discover/who-uses.json]: @kamilmysliwiec thank you for your feedback. I got it

File: content/discover/who-uses.json [hunk 2]
```diff
@@ -171,6 +171,11 @@
       "logo": "/assets/logo/ottonova.png",
       "url": "https://www.ottonova.de",
       "width": "130px"
+    },
+    {
+      "logo": "/assets/logo/radity.png",
+      "url": "https://radity.com",
+      "width": "150px"
```
Comment (kamilmysliwiec) [inline content/discover/who-uses.json:178]:
```suggestion
      "width": "120px"
```

**Comment by kamilmysliwiec** [comment]
Thanks!

**Pull request merged**